src/structures/InputTextComponent.js

```diff
@@ -0,0 +1,201 @@
+'use strict';
+
+const BaseMessageComponent = require('./BaseMessageComponent');
+const { RangeError } = require('../errors');
+const { InputTextStyles, MessageComponentTypes } = require('../util/Constants');
+const Util = require('../util/Util');
+
+/**
+ * Represents an input text component in a modal
+ * @extends {BaseMessageComponent}
+ */
+
+class InputTextComponent extends BaseMessageComponent {
```

Comment:
its a text input, not an input text

Reply:
We'll have to change the v14 PR too then :D